MapboxDirections/OfflineDirections.swift

```diff
@@ -14,20 +14,22 @@ struct AvailableVersionsResponse: Codable {
 public protocol OfflineDirectionsProtocol {
     
     /**
-     Fetches the available versions.
+     Fetches the available routing tile versions. Typically, the most recent version should be passed into `downloadTiles(in:version:completionHandler:)`.
+
+     - parameter completionHandler: Executes when the download completes or fails.
      */
-    @discardableResult
-    func fetchAvailableOfflineVersions(completionHandler: @escaping OfflineVersionsHandler) -> URLSessionDataTask
+    @objc(fetchAvailableOfflineVersionsWithCompletion:)
+    func fetchAvailableOfflineVersions(completionHandler: @escaping OfflineVersionsHandler) -> ()
```

Comment:
I think it’s still useful to return a data task, as the “calculate” methods in Directions do. That way the application can cancel the task if necessary.

Reply:
oh, I missed this comment earlier. does that render this PR (and #352) moot @1ec5 ?

Reply:
No, I think the important thing is that this method resumes the task. It’s fine that it also returns the task.